### PR TITLE
Resolves #264: Add user endpoint method for updating profile pic

### LINKED ISSUE #264: Add user endpoint method for updating profile pic

### PR DESCRIPTION
**Pull request by fradinni**

### CODE REVIEW

File: src/sdk/model/user/user.ts [hunk 1]
```diff
@@ -327,4 +328,13 @@ export class User {
     return Iland.getHttp().delete(`/users/${this.username}/sessions`);
   }
 
+  /**
+   * Updates the user's profile picture.
+   * @param {number} pictureBytes
```
Comment (JoMas971) [inline src/sdk/model/user/user.ts]: The @param is not good. It should be something like : @param {JpegImage} picture

File: src/sdk/model/user/user.ts [hunk 2]
```diff
@@ -327,4 +328,13 @@ export class User {
...
+   * @returns {Promise<void>}
```
Comment (JoMas971) [inline src/sdk/model/user/user.ts]: It should return a Promise<any>

Comment (fradinni) [reply src/sdk/model/user/user.ts]: it returns a promise...

Comment (fradinni) [reply src/sdk/model/user/user.ts]: Iland.get() is a promise

**Pull request merged**